### ISSUE TITLE
Include navi10_get_allowed_feature_mask patch

### ISSUE THREAD

**Issue by dataprolet** [zen-kernel/zen-kernel#312]
Why is the following [patch](https://gitlab.freedesktop.org/drm/amd/-/issues/2196#note_1720712) not included in the kernel? It's included in [linux](https://github.com/torvalds/linux/blob/master/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c) and [archlinux/linux](https://github.com/archlinux/linux/blob/master/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c).

```
diff --git a/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c b/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c
index 0bcd4fe0ef17..5b7a780cbd54 100644
--- a/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/smu11/navi10_ppt.c
@@ -304,7 +304,8 @@ navi10_get_allowed_feature_mask(struct smu_context *smu,
                                | FEATURE_MASK(FEATURE_GFX_SS_BIT)
                                | FEATURE_MASK(FEATURE_APCC_DFLL_BIT)
                                | FEATURE_MASK(FEATURE_FW_CTF_BIT)
-                               | FEATURE_MASK(FEATURE_OUT_OF_BAND_MONITOR_BIT);
+                               | FEATURE_MASK(FEATURE_OUT_OF_BAND_MONITOR_BIT)
+                               | FEATURE_MASK(FEATURE_TEMP_DEPENDENT_VMIN_BIT);
 
        if (adev->pm.pp_feature & PP_SCLK_DPM_MASK)
                *(uint64_t *)feature_mask |= FEATURE_MASK(FEATURE_DPM_GFXCLK_BIT);```

**Comment by damentz** [comment]
Do you have a link to the commit upstream?

**Comment by damentz** [comment]
Fix is in 6.4 branch and was just merged to 6.3/main.
